fulfillments.go

```diff
@@ -127,13 +127,22 @@ type FulfillmentDTO struct {
 
 // ToShopify converts the DTO to the Shopify equivalent
 func (dto FulfillmentDTO) ToShopify() shopify.Fulfillment {
+
+	if dto.CreatedAt.IsZero() {
+		dto.CreatedAt = nil
+	}
+
+	if dto.UpdatedAt.IsZero() {
+		dto.UpdatedAt = nil
+	}
+
 	return shopify.Fulfillment{
 		ID:              dto.ID,
 		OrderID:         dto.OrderID,
 		TrackingNumbers: dto.TrackingNumbers,
 		Status:          dto.Status,
-		CreatedAt:       dto.CreatedAt,
-		UpdatedAt:       dto.UpdatedAt,
+		CreatedAt:       *dto.CreatedAt,
+		UpdatedAt:       *dto.UpdatedAt,
```

Comment:
This can't be right surely? You could be setting CreatedAt/UpdatedAt to nil and then dereferencing them.

Reply:
It was indeed not :( ...